.eslintrc.js

```diff
@@ -242,17 +242,19 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 2018,
   },
-  plugins: ['prettier', 'import'],
-  extends: ['plugin:prettier/recommended'],
```

Comment:
the prettier plugin _enables_ formatter rules, while the config disables it. We want only the latter.